eng/jobs/prepare-signed-artifacts.yml

```diff
@@ -18,23 +18,25 @@ jobs:
 
   - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
     - task: NuGetAuthenticate@0
+      condition: ne(variables['PostBuildSign'], 'true')
```

Comment:
Same for this (comment below)

Reply:
I assumed this was for publishing stuff that shouldn't happen because the repo wouldn't be signing.  Is that misguided?

Reply:
Not positive about this particular call...but the calls windows-build.yml are for sure for internal builds. These are also safe to leave.